tck/src/test/java/org/eclipse/microprofile/jwt/tck/util/TokenUtilsTest.java

```diff
@@ -42,26 +42,52 @@
 public class TokenUtilsTest {
 
     @Test(groups = TCKConstants.TEST_GROUP_UTILS, expectedExceptions = {InvalidJwtException.class},
-            description = "Illustrate validation of iss")
+            description = "Illustrate validation of alg")
     public void testFailAlgorithm() throws Exception {
         Set<TokenUtils.InvalidClaims> invalidFields = new HashSet<>();
         invalidFields.add(TokenUtils.InvalidClaims.ALG);
+        String token = TokenUtils.signClaims("/Token1.json", invalidFields);
+        validateToken(token);
+    }
+    
+    @Test(groups = TCKConstants.TEST_GROUP_UTILS, expectedExceptions = {InvalidJwtException.class},
+            description = "Deprecated: Illustrate validation of alg")
```

Comment:
Why are these just deprecated and not just removed.

Deprecated doesn't make sense for TCK IMHO.

Reply:
@rdebusscher I've seen `TokenUtils` being used outside of TCK, certainly our additional product tests refer to it as far as I recall. It is very feasible someone else, even the users, try them since it is the only support currently available around MP JWT itself to generate the token.